CONTRIBUTING.md

```diff
@@ -5,10 +5,13 @@ description of best practices for developing scientific packages.
 
 # Setting up a development environment
 
-You can set up a development environment by running:
+You can set up a development environment with `conda` or your environment
+manager of choice:
 
 ```bash
-conda env create -f environment.yml
```

Comment:
can we delete the `environment.yml` now?

Reply:
I think we should, yeah :) I was not sure your thoughts.

Reply:
a210a07